packages/graph/hash_graph/lib/graph/src/knowledge/link/mod.rs

```diff
@@ -67,8 +68,8 @@ pub enum Outgoing {
 
 /// A collection of links that originate from the same source entity.
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize, Component)]
+#[serde(rename_all = "camelCase")]
 pub struct Links {
-    #[serde(flatten)]
```

Comment:
Why did we get rid of this

Reply:
Was this to do with the OpenAPI library not being able to deal with flatten or something?

Reply:
Yes, exactly

Reply:
@TimDiekmann any concerns about this?

Reply:
As long as the tests are passing I don't mind changing the `serde` attributes 🚀

Reply:
Well we've updated the tests here so that might not be the best criteria 😁 

Is there a reason we have the `outgoing` field? Could we perhaps change it to an untagged new-type to avoid the flatten?

`OutgoingLinks(HashMap<...`?

Or did we intend to add more fields

Reply:
If we intend to have `incoming` as well, it would make sense for this struct to be as it is. I don't remember how utoipa handles newtypes that are not just strings, but it might have difficulties when it's a `HashMap`

Reply:
Would we put `incoming` in here? 
And if there are other solutions that we can explore I think we should. I'd prefer to avoid polluting these structs if we can.

Reply:
I suppose we wouldn't put it in here, as it might be a structural query result to get all incoming links. Do you want me to see what happens if we use a newtype instead?

Reply:
I think if we are not planning on adding `incoming` here (which would probably be odd), avoiding the `.outgoing` would be good I guess.